cmd/mistry/main.go

```diff
@@ -67,29 +68,34 @@ JOB PARAMETERS:
    -- dynamic options for the command
 
 EXAMPLES:
-   1. The following sequence schedules a command to be executed in http://example.org:9090,
-      builds the yarn project, groups it under group_name, syncs the result to the
-      directory /tmp/yarn and uses the dynamic arguments yarn.lock and foo.
-      Note the @ before the yarn.lock, this indicates a referral to an actual file on the
-      filesystem.
+    1. Talk to the mistry service at http://example.org:9090, to build the yarn
+       project, group it under group_name, sync the result to the
+       directory /tmp/yarn and use the dynamic arguments yarn.lock and foo.
+       Note the @ before the yarn.lock, this indicates a referral to an actual file on the
+       filesystem.
 
-      $ mistry-cli build --host example.org --port 9090 --project yarn \
-      --group group_name --target /tmp/yarn -- --lockfile=@yarn.lock --foo=bar
+       $ {{.HelpName}} --host example.org --port 9090 --project yarn \
+	     --group group_name --transport rsync --target /tmp/yarn \
+	     -- --lockfile=@yarn.lock --foo=bar
 
-   2. The following sequence uses the GROUP environment variable to set the project group.
+    2. Uses the GROUP environment variable to set the project group.
 
-      $ GROUP=group_name mistry-cli build --host example.org --port 9090 --project yarn \
-      --target /tmp/yarn -- --lockfile=@yarn.lock --foo=bar
-	`, cli.CommandHelpTemplate)
+	   $ GROUP=group_name {{.HelpName}} --host example.org --port 9090 --project yarn \
+	     --transport rsync --target /tmp/yarn -- --lockfile=@yarn.lock --foo=bar
+
+    3. Schedules a build and exits early without waiting for the result by not
+	   specifying a transport
+
+       $ {{.HelpName}} -host example.org --port 9090 --project yarn
+`, cli.CommandHelpTemplate)
 
 	app := cli.NewApp()
-	app.Name = "mistry-cli"
```

Comment:
Is the `.Name` set automatically by urfave/cli?

Reply:
yes, `os.Args[0]`

https://github.com/urfave/cli/blob/3fb51f15e8ce6ea8428a8dacb30d120fad8fa186/app.go#L83